manifests/init.pp

```diff
@@ -185,14 +194,16 @@
         $config_file_v6_content = template($config_template_v6)
       }
 
+      assert_type(Stdlib::Absolutepath, $v6_path)
```

Comment:
this `assert_type()` seems entirely unnecessary, but the `file` should be conditional, since `$v6_path` is `Optional`